server/src/infra/tcp_server.rs

```diff
@@ -110,12 +114,9 @@ fn http_config<Backend>(
             .configure(super::graphql::api::configure_endpoint::<Backend>),
     )
     .service(
-        web::resource("/pkg/lldap_app_bg.wasm").route(
-            web::route()
-                .wrap(middleware::Compress::default())
```

Comment:
I think the compress middleware was not doing anything since I was serving an already-compressed file.